docs/sources/release-notes/v1-4.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Version 1.4 release notes
+menuTitle: V1.4
+description: Release notes for Grafana Pyroscope 1.4
+weight: 750
+---
+
+# Version 1.4 release notes
+
+We are excited to present Grafana Pyroscope 1.4.
+
+This release includes several new features will are precursors to larger projects like:
+
+* Adhoc profiling
+* Time series scoped to a flamegraph function
+* Traces to profiles tooling
+
+Additionally, there has been a multitude of smaller changes to improve stability, performance, and documentation.
```

Comment:
Multitude sounds odd in this sentence. What do you think about this? 
```suggestion
Additionally, numerous other changes improve stability, performance, and documentation.
```

Reply:
Your suggestion has better flow and is more concise. I like it 👍